package.json

```diff
@@ -53,8 +53,8 @@
 		"lint:js:report": "npm run lint:js -- --output-file eslint_report.json --ext=js,ts,tsx --format json",
 		"lint:js-fix": "eslint assets/js --ext=js,jsx,ts,tsx --fix",
 		"lint:md:docs": "wp-scripts lint-md-docs",
-		"lint:php": "composer run-script phpcs ./src",
-		"lint:php-fix": "composer run-script phpcbf ./src",
+		"lint:php": "composer run-script phpcs ./src && composer run-script phpcs ./tests/mocks/woo-test-helper",
+		"lint:php-fix": "composer run-script phpcbf ./src && composer run-script phpcbf ./tests/mocks/woo-test-helper",
```

Comment:
I've updated these entries so that the `woo-test-helper` plugin can be linted and fixed.